lib/util.js

```diff
@@ -1159,21 +1168,11 @@ module.exports = {
     if (responseBodyWrapper.contentTypeHeader) {
       // we could infer the content-type header from the body
       responseHeaders.push({ key: 'Content-Type', value: responseBodyWrapper.contentTypeHeader });
-      if (responseBodyWrapper.contentTypeHeader === APP_JSON) {
-        previewLanguage = 'json';
-      }
-      else if (responseBodyWrapper.contentTypeHeader === APP_XML) {
-        previewLanguage = 'xml';
-      }
+      previewLanguage = this.getContentTypeHeaderFormat(responseBodyWrapper.contentTypeHeader);
```

Comment:
@abhijitkane how about this?

Reply:
Nope - previewLanguage being the same as the header family is just a coincidence. Let's define a separate enum for previewLanguage, and explicitly set it to json if the header family is json.
or, we write a function to convert header family to previewLanguage